tests/test_globalization.py

```diff
@@ -232,4 +297,16 @@ def get_next_characters(self, all_characters, count=50):
             next_characters_length = min(len(all_characters), count)
             next_characters = all_characters[0:next_characters_length]
             all_characters = all_characters[next_characters_length:]
-            yield next_characters
\ No newline at end of file
+            yield next_characters
+    
+    def get_all_characters(self):
```

Comment:
Not a fan of this change since when something fails, we don't know which charset is the problem. Can you add some logging of the charset name as we're testing? Or, just create a test per charset like before?

Reply:
fixed to test per charset like before.